src/main/java/picard/analysis/TheoreticalSensitivityMetrics.java

```diff
@@ -0,0 +1,33 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+/**
+ * Created by fleharty on 6/30/17.
+ */
+public class TheoreticalSensitivityMetrics extends MetricBase {
```

Comment:
wouldn't it be better to just emit a histogram with the values you want? it would be much easier to extend to arbitrary values. You could then also have the list of desired values as an input parameter to the CLP...